config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -394,6 +394,45 @@ presubmits:
             limits:
               cpu: "7"
               memory: "10Gi"
+  - name: pull-kueue-test-e2e-kueueviz-main
+    cluster: eks-prow-build-cluster
+    branches:
+      - ^main
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$|^(keps|site|charts)/"
+    decorate: true
+    path_alias: sigs.k8s.io/kueue
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: pull-kueue-test-e2e-certmanager-main
+      description: "Run cert-manager end to end tests"
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250227-3a13bdd784-master
+          env:
+            - name: E2E_KIND_VERSION
+              value: kindest/node:v1.32.3
+            - name: BUILDER_IMAGE
+              value: public.ecr.aws/docker/library/golang:1.24
+            - name: CYPRESS_IMAGE_NAME
+              value: cypress/base:22.14.0
```

Comment:
Actually, it is better to configure the specific cypress image inside Kueue.

This way we can test before merging.

Reply:
ok, got it, removed from here.
it is configured kueue with the latest changes.